### DISCUSSION TITLE
DiehlAndCook2015 network model parameters

### DISCUSSION THREAD

**Discussion by rafaelblevin821** [BindsNET/bindsnet#640] [General]
Hi, I was wondering if there is any reason why `thresh` is not a callable argument in the `DiehlAndCook2015` model? 

I want to try different values for `thresh` to see how it affects the training and classification accuracy. I tried adding it to the `DiehlAndCook2015` model and added the argparse for it, but I get an error message telling me it is not a callable argument.

Any info on this please?

**Comment by Hananel-Hazan** [comment]
Hi, I added the threshold parameters for all existing models #638.
